Strongly indicate that this is legacy software. (#331)
signalfx/signalfx-java: README.md

<p align="center">
  <a href="https://github.com/signalfx/signalfx-java/actions?query=workflow%3A%22CI+build%22">
    <img alt="Build Status" src="https://github.com/signalfx/signalfx-java/actions/workflows/ci.yaml/badge.svg">
  </a>
</p>

---

# SignalFx client libraries

This repository contains legacy libraries for instrumenting Java applications and
reporting metrics to Splunk Observability Cloud (formerly SignalFx).

> Splunk has adopted OpenTelemetry and users are strongly encouraged to use the 

Reviewer: ```suggestion
Splunk has adopted OpenTelemetry. Use the 
```

[OpenTelemetry Java SDK](https://github.com/open-telemetry/opentelemetry-java) or
the [Splunk Distribution of OpenTelemetry Java Instrumentation](https://github.com/signalfx/splunk-otel-java)

> agent to send telemetry data to Splunk. Users should *NOT* be integrating `signalfx-java` 

Reviewer: ```suggestion
agent to send telemetry data to Splunk. Do not integrate `signalfx-java` 
```

into new services.

> To send data to Splunk, you will need a Splunk Observability account and organization

Reviewer: ```suggestion
To send data to Splunk, you need a Splunk Observability account and organization
```

Reviewer: Send data... using this lib?

Author: Sure, or any lib really. Can't ingest data without an ingest token.

API token to use them. For more information on SignalFx and to create an
account, go to [Splunk Observability](https://www.splunk.com/en_us/products/observability.html).

## Language compatibility

> * Java 8+

Reviewer: ```suggestion
* Java 8 and higher
```

## Using this library in your project

> Users are discouraged from integrating this library into new services. Please 

Reviewer: ```suggestion
You are discouraged from integrating this library into new services. 
```

> use [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-java) to send data 

Reviewer: ```suggestion
Use [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-java) to send data 
```

<p align="center">
  <a href="https://github.com/signalfx/signalfx-java/actions?query=workflow%3A%22CI+build%22">
    <img alt="Build Status" src="https://github.com/signalfx/signalfx-java/actions/workflows/ci.yaml/badge.svg">
  </a>
</p>

---

# SignalFx client libraries

This repository contains legacy libraries for instrumenting Java applications and
reporting metrics to Splunk Observability Cloud (formerly SignalFx).

Splunk has adopted OpenTelemetry. Use the 
[OpenTelemetry Java SDK](https://github.com/open-telemetry/opentelemetry-java) or
the [Splunk Distribution of OpenTelemetry Java Instrumentation](https://github.com/signalfx/splunk-otel-java)
agent to send telemetry data to Splunk. Do not integrate `signalfx-java` 
into new services.

> To send data to Splunk, you need a Splunk Observability account and organization

Reviewer: ```suggestion
To send data to Splunk, you need a Splunk Observability Cloud account and organization
```

Author: I'm not going to say what I really think of this. Heh. 🙊

> API token to use them. For more information on SignalFx and to create an

Reviewer: ```suggestion
API token. For more information on Observability Cloud and to create an
```
